vol_test.c

```diff
@@ -41,13 +41,17 @@
 
 char vol_test_filename[VOL_TEST_FILENAME_MAX_LENGTH];
 
+char *test_path_prefix = "";
+
 size_t n_tests_run_g;
 size_t n_tests_passed_g;
 size_t n_tests_failed_g;
 size_t n_tests_skipped_g;
 
 uint64_t vol_cap_flags;
 
+char prefixed_filename[VOL_TEST_FILENAME_MAX_LENGTH + VOL_TEST_PATH_MAX_LENGTH];
```

Comment:
Generally we should prefer to dynamically allocate buffers like this. If they get too large GCC will complain about the stack usage and eventually that will fail the -Werror check when we merge the VOL tests back into the library.